src/styles.scss

```diff
@@ -1,5 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
 @import "~@ng-select/ng-select/themes/default.theme.css";
+@import "~bootstrap/dist/css/bootstrap.css";
```

Comment:
Are you sure it's OK? Usually you should get styles from external libraries using angular.json.

Reply:
Please look at - https://github.com/angular/angular-cli/issues/9690